src/ui-fragment/src/components/ui-serach-input.tsx

```diff
@@ -31,18 +34,33 @@ export const UISearchInput = ({ state, setState, name }: {
 		return <></>;
 	}
 
-	return <Search
-		id={state.codeContext?.name}
+	return state.expandable
+		? <ExpandableSearch
+			size={state.inputSize}
+			disabled={state.disabled}
+			autoComplete={state.autoComplete}
```

Comment:
```suggestion
			autoComplete={state.autocomplete}
```

Also change line 10 in this file to use lowercase c so:
```
autocomplete?: string;
```